Dockerfile

```
@@ -0,0 +1,47 @@
+# This builds Boulder in a Docker container, then creates an image
+# containing just the built Boulder binaries plus some ancillary
+# files that are useful for predeployment testing.
+FROM docker.io/ubuntu:24.04 AS builder
+
+ARG COMMIT_ID
+ARG GO_VERSION
+ARG VERSION
+
+ENV DEBIAN_FRONTEND=noninteractive
+RUN apt-get --assume-yes --no-install-recommends --update install \
+    ca-certificates curl gcc git gnupg2 libc6-dev
+
+COPY tools/fetch-and-verify-go.sh /tmp
+RUN /tmp/fetch-and-verify-go.sh ${GO_VERSION}
+RUN tar -C /opt -xzf go.tar.gz
+
+COPY . /opt/boulder
+WORKDIR /opt/boulder
+
```

Optional: You can go to the path here and just use `go install` on line `23` below.

```suggestion

ENV PATH="/opt/go/bin:${PATH}" 
```